docs/src/how-do-we-build-an-index.md

```diff
@@ -5,54 +5,57 @@ title: How do we build a DocSearch index?
 
 In this section you will learn how we build a DocSearch index from your page.
 
-### Everything starts from your page
+## Everything starts from your page
 
 ![1st step][2] {mt-2}
 
-_Note: We would rather avoid useless js rendering. All useful information should
-be available without any client-side rendering. You can use `curl` in order to
-have a better idea of how it would look without any client-side rendering._
+_Note: We would rather avoid useless JavaScript rendering. All useful
```

Comment:
I feel this comment is confusing. What is the intent behind it? What do you want to tell the reader? As a reader, I'm not sure what I should do with this information and I don't see how it makes me better understand how the crawler works.